recipe/conda_build_config.yaml

```diff
@@ -16,11 +16,8 @@ m2w64_c_stdlib:                # [win]
   - m2w64-sysroot              # [win]
 m2w64_c_stdlib_version:        # [win]
   - 12                         # [win]
-c_stdlib_version:              # [unix]
-  - 2.17                       # [linux]
-  - 2.17                       # [linux and not ppc64le and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - 10.13                      # [osx and x86_64]
-  - 11.0                       # [osx and arm64]
+c_stdlib_version:   # [osx]
+  - 11.0            # [osx]
```

Comment:
@conda-forge/core, please revert this change. I believe it was made in error. There is no documented reason why closing a migration for absl would remove channel-pinnings for the c_stdlib_version on non-osx platforms.

Reply:
Can you send a PR?